utils/constants.py

```diff
@@ -397,11 +397,30 @@
                         "RewarderTokenDecimals": 18,
                     }
                 }
-            }
+            },
+        28: {
+            "LP": "0x87BCC091d0A7F9352728100268Ac8D25729113bB",
+            "LPType": "StableAMM",
+            "Rewarder": ZERO_ADDRESS,
+            "CoingeckoRewarderTokenName": "",
+            "RewarderPriceLP": "",
+            "RewarderTokenDecimals": "",
+            "Rewarders": {
+                    0: {
+                        "Rewarder": ZERO_ADDRESS,
+                        "CoingeckoRewarderTokenName": "",
+                        "RewarderPriceLP": "",
+                        "RewarderTokenDecimals": "",
+                    }
+                }
+            },
     }
 
 V2_STABLEPOOL_FACTORY = {
         18: {
             "poolContract": "0x13e7a001EC72AB30D66E2f386f677e25dCFF5F59"
```

Comment:
@baboobhaiya does the new format work with stable AMMs? Just want to confirm. Looks good otherwise

Reply:
Yes approving the PR